content/knowledge-base/lifecycle.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Product Lifecycle
+---
+
+## Software List
+
+Upbound provides support and software maintenance to paid customers. Supported software products include:
+
+* [UXP]({{< ref "uxp/_index.md" >}})
+* [Official Providers]({{< ref "upbound-marketplace/providers/_index.md" >}})
+
+## Maintenance and Updates
+
+Upbound supports Generally Available releases for up to 12 months. 
+All software follows semantic versioning of `X.Y.Z`.
+
+* A major release is a change in the first digit and usually indicates breaking API changes or, in the case of 1.0.0, indicates the first release where the API is considered complete (-enough) and stable.
+* A minor release is a change in the second digit and usually indicates new features added, and usually also includes bug fixes that might not have been released yet in a previous patch release.
```

Comment:
```suggestion
* A minor release is a change in the second digit and indicates new features were added. It could also include new bug fixes that were not included in a previous patch release.
```